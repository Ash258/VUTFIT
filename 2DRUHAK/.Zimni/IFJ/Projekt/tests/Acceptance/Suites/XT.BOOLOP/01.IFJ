' Prime number

function prvocislo(a as integer) as boolean
    dim delitel as integer = a - 1

    do while delitel > 1
        if ((a \ delitel) * delitel) = a then
            return false
        else
        end if
        delitel = delitel - 1
    loop

    return true
end function

scope
    dim cislo as integer = 13
    dim overeni as boolean
    overeni = prvocislo(cislo)
    if overeni then
        print !"Cislo "; cislo ; !" je prvocislo";
    else
        print !"Cislo "; cislo ; !" neni prvocislo";
    end if
end scope
