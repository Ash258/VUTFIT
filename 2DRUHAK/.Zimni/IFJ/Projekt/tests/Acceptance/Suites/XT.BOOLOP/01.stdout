Cislo  13 je prvocislo